### PR TITLE
[655] Remove Cypress tests

### PR DESCRIPTION
**Pull request by gpeng**
### Context

The Cypress tests are end-to-end tests that are run as part of CI. These are failing currently and are a pain to maintain.

We are implementing some lighter weight rspec based smoke tests to replace these in the deployment pipeline.

### Changes proposed in this pull request

Remove them.

### Checklist

- [x] Make sure all information from the Trello card is in here
- [x] Attach to Trello card
- [x] Rebased master
- [x] Cleaned commit history
- [ ] Tested by running locally
- [ ] Product Review

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -191,45 +191,6 @@ the server:
...
-    # using $PWD/end-to-end-tests/config/local.json
-    yarn run cy:run
-    ```
-
-    ``` bash
-    # native
-    yarn run cy:run --env 'email=someone@test.com,password=change me'
-    ```
-
 ### Additional required setup
```
Comment (defong) [inline README.md:194]: All this can go as well

**Review by defong — APPROVED** [review]
Need to delete more

**Review by defong — CHANGES_REQUESTED** [review]
Actually there is more to delete
https://github.com/DFE-Digital/publish-teacher-training/blob/57089ce594c1fd14a8baa7685c486e6b96303b7f/azure-pipelines.yml#L149

**Comment by gpeng** [comment]
[Remove Cypress tests](https://trello.com/c/HF9cnssK/875-remove-cypress-tests)

**Comment by gpeng** [comment]
> Need to delete more

Half a job 🙄 I've removed those bits. Should be good now 👍

**Review by defong — APPROVED** [review]
Good work!

**Pull request merged**